infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftProject.java

```diff
@@ -127,11 +140,64 @@ private void create(String projectName, OpenShiftClient client) throws Infrastru
           .withName(projectName)
           .endMetadata()
           .done();
+      waitServiceAccountReadiness(projectName, client);
```

Comment:
Looks like this case is not covered by unit tests. WDYT?

Reply:
Yes, sure I'll add them.